lua/lazyvim/plugins/lsp/keymaps.lua

```diff
@@ -5,6 +5,8 @@ M._keys = nil
 
 ---@return (LazyKeys|{has?:string})[]
 function M.get()
+  local telescope = require("telescope.builtin")
```

Comment:
Please inline the require, otherwise Telescope will always be loaded whenever an lsp attaches

Reply:
👌